src/utils/stringToBoolean.js

```diff
@@ -1,5 +1,5 @@
 import isString from "./isString";
 
 export default str => {
-  return isString(str) && (str.toLowerCase() === "true" || str === "1");
+  return isString(str) && str.toLowerCase() === "true";
```

Comment:
I felt like allowing a string of "1" was a too generous and if we're ever allowing `1` to signify `true` then we should probably reconsider why we're allowing it. We don't seem to be supporting it anywhere (I think it was considered for the `alloy_debug` query string parameter at one point but it's not documented as supported).